.github/workflows/bump_poetry.yml

```diff
@@ -29,10 +21,10 @@ jobs:
         run: |
           git config user.name github-actions
           git config user.email github-actions@github.com
```

Comment:
For my understanding: this file only involves automatic bumps upon merge commits. Correct? In that case it is correct and should only bump the patch version.

Updating minor/major is then always a manual action -> agreed?

Reply:
correct & agreed